packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -169,6 +179,16 @@ class VelocityTracker {
   ///
   /// Returns null if there is no data on which to base an estimate.
   VelocityEstimate? getVelocityEstimate() {
+    // Has user recently moved since last sample?
+    if (_sinceLastSample.elapsedMilliseconds > VelocityTracker._assumePointerMoveStoppedMilliseconds) {
```

Comment:
Do we have style guide around using static member in the same class? I think in other class we don't include the class name

Reply:
The analyzer required it